src/FastRandom.cpp

```diff
@@ -13,109 +13,49 @@
 #elif defined (_MSC_VER)
 	#define ATTRIBUTE_TLS static __declspec(thread)
 #else
-	#error "Unknown thread local storage qualifier"
+	#define ATTRIBUTE_TLS thread_local
```

Comment:
Oh, it's already standard. Then we should probably remove the `#ifdef` altogether.